lib/Runtime/Library/JavascriptLibrary.cpp

```diff
@@ -924,7 +978,7 @@ namespace Js
 
     DynamicType * JavascriptLibrary::CreateDeferredPrototypeFunctionType(JavascriptMethod entrypoint)
     {
-        return CreateDeferredPrototypeFunctionTypeNoProfileThunk(this->inDispatchProfileMode ? ProfileEntryThunk : entrypoint);
+        return CreateDeferredPrototypeFunctionTypeNoProfileThunk(this->inDispatchProfileMode ? ProfileEntryThunk : entrypoint, true);
```

Comment:
FINALLY got some real time to look at this, and this line was the problem for the chakrafull jsrt test. Do you remember what the goal was here? It doesn't seem related to the rest of the PR, and reverting the line causes the tests to pass (don't feel the need to make the change yourself if its ok to revert, I will push my own that's easier to sync with the chakrafull test changes I have already made).

Reply:
This change should be reverted.

Reply:
I think we can leave giving these lengths turned off for now and test it when development ramps back up for the next windows release (since these include projected and custom external functions, we would probably have to make the change based on the results of our chakrafull test suite, so that will be on us). As for the name, it may be confusing, but since the goal is to have lengths be enabled by default everywhere in the future, I think its fine to be in this state for now (especially since it was already inconsistent before, so its not a regression). @pleath sounds like a plan?

Reply:
I'm fine with reverting the one line, pushing what we have, and adjusting the names as a separate PR, if that's what you're suggesting.